test/liquid_test.rb

```diff
@@ -5,7 +5,11 @@
 require 'test_helper'
 require 'liquid/c'
 
-test_files = FileList[File.join(liquid_test_dir, '{integration,unit}/**/*_test.rb')]
+test_files = FileList[File.join(liquid_test_dir, 'integration/**/*_test.rb')]
+
+# Test the Variable#strict_parse patch
+test_files << File.join(liquid_test_dir, 'unit/variable_unit_test.rb')
```

Comment:
Without this, tests doing things like `assert_equal(Variable, template.root.nodelist[0].class)` would fail because we aren't using the Liquid::Variable class for the variable nodes, even if we otherwise provided full duck type compatibility with Liquid::Variable.

There are also a number of unit tests that aren't really relevant to liquid-c, since they are using internal classes directly rather than going through methods monkey patched by liquid-c.  Those tests would be harmless to run, but would just give a false sense of having tested more things.